test/functional/aws_shared_functions.go

```diff
@@ -26,39 +27,45 @@ const (
 	awsCredsSecretName = "aws-creds"
 )
 
-var (
+func getExpectedPostgres(installationName string) []string {
```

Comment:
Should we have some logic in this function to check the install type and return a different list depending on it?

Reply:
I can confirm that this is an issue for Managed API nightly pipeline. The test fails like this:
```--- FAIL: TestIntegreatly/Integreatly_Happy_Path_Tests/F01_-_Verify_AWS_rds_resources_exist_and_are_in_expected_state (2.85s)
            aws_rds_resources_exist.go:20: test cro postgres exists failed with the following errors : [
                failed to find codeready-postgres-rhmi postgres cr : postgres.integreatly.org "codeready-postgres-rhmi" not found 
                found codeready-postgres-rhmi postgres not ready with phase: , message:  
                codeready-postgres-rhmi postgres cr does not contain a resource id annotation: annotations for  can not be nil 
                failed to find ups-postgres-rhmi postgres cr : postgres.integreatly.org "ups-postgres-rhmi" not found 
                found ups-postgres-rhmi postgres not ready with phase: , message:  
                ups-postgres-rhmi postgres cr does not contain a resource id annotation: annotations for  can not be nil 
                failed to find fuse-postgres-rhmi postgres cr : postgres.integreatly.org "fuse-postgres-rhmi" not found 
                found fuse-postgres-rhmi postgres not ready with phase: , message:  
                fuse-postgres-rhmi postgres cr does not contain a resource id annotation: annotations for  can not be nil]```

Reply:
This failure is due to very same reason:
```--- FAIL: TestIntegreatly/Integreatly_Happy_Path_Tests/A21_-_Verify_AWS_maintenance_and_backup_windows (307.94s)```

Reply:
Code updated to address this